content/en/docs/_index.md

```diff
@@ -8,12 +8,12 @@ draft: false
 Welcome to the {{< mtf-wiki >}}, a place that may help some transgender women.
 
 {{< notice info >}}
-Thought this site was founded initially for Chinese trans, we provide this English page to show our visibility. However, if you would like to contribute information for trans in your region, welcome to do it.
+Though this site was founded initially for Chinese trans, we provide this English page to show our visibility. However, if you would like to contribute information for trans in your region, welcome to do it.
```

Comment:
... for Chinese transgender people ... for transgender people in your region, it is always welcome to do it.